crates/rnote-ui/data/ui/mainheader.ui

```diff
@@ -59,7 +59,7 @@
           </object>
         </child>
         <child type="end">
-          <object class="GtkBox">
+          <object class="GtkBox" id="right_buttons">
```

Comment:
I'd name the box `right_buttons_box` for consistency

Reply:
fixed